tensorflow/lite/ios/BUILD.apple

```diff
@@ -84,6 +84,7 @@ strip_common_include_path_prefix(
     name = "strip_common_include_path_core",
     hdr_labels = [
         "//tensorflow/lite/c:c_api.h",
+        "//tensorflow/lite/c:c_api_experimental.h",
```

Comment:
@yyoon Is this fine? or should there be a different BUILD target with this experimental header (I am not very familiar with iOS BUILDs)

Reply:
Looks fine to me. We actually had a separate issue filed for adding C experimental API:
https://github.com/tensorflow/tensorflow/issues/46852